src/extension/qrCodeContentProvider.ts

```diff
@@ -2,13 +2,13 @@
 // Licensed under the MIT license. See LICENSE file in the project root for details.
 
 import * as qr from "qr-image";
-import { TextDocumentContentProvider, Uri, CancellationToken } from "vscode";
```

Comment:
Please clarify why did you make this change? Is the CancellationToken is redundant?

Reply:
Yes, it's not used now